aws/policy/application-services.yaml

```diff
@@ -130,6 +130,13 @@ Statement:
       - logs:CreateLogGroup
       - logs:DeleteLogGroup
     Resource: arn:aws:logs:us-east-1:{{ aws_account_id }}:log-group:ansible-test*
+  - Sid: ModifyCloudwatchAlarms
+    Effect: Allow
+    Action:
+      - cloudwatch:PutMetricAlarm
+      - cloudwatch:DeleteAlarms
+      - cloudwatch:DescribeAlarms
```

Comment:
`cloudwatch:DescribeAlarms` has to be able to operate on `Resource: arn:aws:cloudwatch:us-east-1:{{ aws_account_id }}:alarm:*` to retrieve names. Could you please move this action to a `DescribeCloudwatchAlarms` sid that operates on that resource?

Reply:
Good catch.  Thanks for the review